pkg/ccl/changefeedccl/changefeed_dist.go

```diff
@@ -345,37 +345,47 @@ func startDistChangefeed(
 // The bin packing choice gives preference to leaseholder replicas if possible.
 var replicaOracleChoice = replicaoracle.BinPackingChoice
 
-type rangeDistributionType int
+type clusterSettingRangeDistributionType int
 
 const (
 	// defaultDistribution employs no load balancing on the changefeed
 	// side. We defer to distsql to select nodes and distribute work.
-	defaultDistribution rangeDistributionType = 0
+	defaultDistribution clusterSettingRangeDistributionType = 0
 	// balancedSimpleDistribution defers to distsql for selecting the
 	// set of nodes to distribute work to. However, changefeeds will try to
 	// distribute work evenly across this set of nodes.
-	balancedSimpleDistribution rangeDistributionType = 1
+	balancedSimpleDistribution clusterSettingRangeDistributionType = 1
 	// TODO(jayant): add balancedFullDistribution which takes
 	// full control of node selection and distribution.
 )
 
-// RangeDistributionStrategy is used to determine how the changefeed balances
-// ranges between nodes.
-// TODO: deprecate this setting in favor of a changefeed option.
+var rangeDistributionStrategyStrings = map[clusterSettingRangeDistributionType]string{
+	defaultDistribution:        "default",
+	balancedSimpleDistribution: "balanced_simple",
+}
+
 var RangeDistributionStrategy = settings.RegisterEnumSetting(
 	settings.ApplicationLevel,
 	"changefeed.default_range_distribution_strategy",
 	"configures how work is distributed among nodes for a given changefeed. "+
 		"for the most balanced distribution, use `balanced_simple`. changing this setting "+
-		"will not override locality restrictions",
+		"will not override locality restrictions. this setting can be overridden by the "+
+		"changefeed option `range_distribution_strategy`",
 	metamorphic.ConstantWithTestChoice("default_range_distribution_strategy",
 		"default", "balanced_simple"),
-	map[rangeDistributionType]string{
-		defaultDistribution:        "default",
-		balancedSimpleDistribution: "balanced_simple",
-	},
+	rangeDistributionStrategyStrings,
 	settings.WithPublic)
 
+func rangeDistributionStrategyToString(strategy clusterSettingRangeDistributionType) string {
```

Comment:
I think we can just directly convert from the enum to `changefeedbase.ChangefeedRangeDistributionStrategy`